Source/missiles.cpp

```diff
@@ -2878,9 +2878,9 @@ void MI_Fireball(int i)
 			SetMissAnim(i, MFILE_BIGEXP);
 			missile[i]._mirange = missile[i]._miAnimLen - 1;
 		} else if (missile[i]._mix != missile[i]._miVar1 || missile[i]._miy != missile[i]._miVar2) {
-			missile[i]._miVar1 = missile[i]._mix;
-			missile[i]._miVar2 = missile[i]._miy;
-			ChangeLight(missile[i]._mlid, missile[i]._mix, missile[i]._miy, 8);
+				missile[i]._miVar1 = missile[i]._mix;

```

Comment:
why the extra indentation :)

Reply:
> why the extra indentation :)

The formatting tools also gotta feel like they are contributing towards a common goal :)

Reply:
fixed :P